phd-tests/tests/src/crucible/smoke.rs

```diff
@@ -88,8 +88,9 @@ async fn shutdown_persistence_test(ctx: &Framework) {
 }
 
 #[phd_testcase]
-async fn vcr_replace_test(ctx: &Framework) {
-    let mut config = ctx.vm_config_builder("crucible_vcr_replace_test");
```

Comment:
by replacing the previous VCR replacement test where the VM has already started, are we no longer testing some code paths? would it make sense to have kept the prior test as well as adding the new one for replacement during a start?

Reply:
I updated the test so that it attempts a second replacement after the VM starts running. That way we only have to wait for one guest boot sequence (and we also verify that we can replace VCRs more than once on a single instance).